interfaces/cython/cantera/onedim.py

```diff
@@ -1280,28 +1280,28 @@ def solve(self, loglevel=1, refine_grid=True, auto=False):
         if loglevel > 0:
             if self.extinct():
                 print('WARNING: Flame is extinct.')
-
-            # Check if the flame is very thick
-            # crude width estimate based on temperature
-            z_flame = self.grid[self.T > np.max(self.T) / 2]
-            flame_width = z_flame[-1] - z_flame[0]
-            domain_width = self.grid[-1] - self.grid[0]
-            if flame_width / domain_width > 0.4:
-                print('WARNING: The flame is thick compared to the domain '
-                      'size. The flame might be affected by the plug-flow '
-                      'boundary conditions. Consider increasing the inlet mass '
-                      'fluxes or using a larger domain.')
-
-            # Check if the temperature peak is close to a boundary
-            z_center = (self.grid[np.argmax(self.T)] - self.grid[0]) / domain_width
-            if z_center < 0.25:
-                print('WARNING: The flame temperature peak is close to the '
-                      'fuel inlet. Consider increasing the ratio of the '
-                      'fuel inlet mass flux to the oxidizer inlet mass flux.')
-            if z_center > 0.75:
-                print('WARNING: The flame temperature peak is close to the '
-                      'oxidizer inlet. Consider increasing the ratio of the '
-                      'oxidizer inlet mass flux to the fuel inlet mass flux.')
+            else:
+                # Check if the flame is very thick
+                # crude width estimate based on temperature
+                z_flame = self.grid[self.T > np.max(self.T) / 2]
+                flame_width = z_flame[-1] - z_flame[0]
+                domain_width = self.grid[-1] - self.grid[0]
+                if flame_width / domain_width > 0.4:
+                    print('WARNING: The flame is thick compared to the domain '
+                          'size. The flame might be affected by the plug-flow '
+                          'boundary conditions. Consider increasing the inlet mass '
+                          'fluxes or using a larger domain.')
+
+                # Check if the temperature peak is close to a boundary
+                z_center = (self.grid[np.argmax(self.T)] - self.grid[0]) / domain_width
+                if z_center < 0.25:
+                    print('WARNING: The flame temperature peak is close to the '
+                          'fuel inlet. Consider increasing the ratio of the '
+                          'fuel inlet mass flux to the oxidizer inlet mass flux.')
```

Comment:
```suggestion
                    warnings.warn("The flame temperature peak is close to the "
                          "fuel inlet. Consider increasing the ratio of the "
                          "fuel inlet mass flux to the oxidizer inlet mass flux.")
```
Could you replace the regular print statements by warnings here and in the other lines that are touched by this PR? (the default `UserWarning` is likely the best option). Also, in all new/modified code, single quotes should be replaced by double quotes.

Reply:
Ugh. Walked away for a minute. Nevermind ...

Reply:
see #1339